#7823 Make Contact component handle multiple institutions

aliceh75 opened the pull request:
It is now possible to select multiple institutions for a contact in the Drupal backend.

This PR updates the Contact component to accept an array of institutions, rather than a single one.

Note I didn't feel it worthwhile to create an &lt;Institution&gt; component, as this is the only place this is used (and likely if it was used elsewhere, it would be displayed differently).

Matching changes to corporate-react are done and will come once this is merged & published.

aliceh75 committed:
#7823 Make Contact component handle multiple institutions

creido-welly commented on src/components/Contact/Contact.tsx:
```diff
@@ -67,13 +70,12 @@ export const Contact = ({
-        </p>
-      )}
+      {institutions &&
+        institutions.map(e => (
```
Minor: you could use object destructuring here to access the keys directly i.e.

```js
institutions.map(({ country, name }) => (
  <p className="cc-contact__institution" itemProp="worksFor">
    {country ? `${name}, ${country}` : name}
  </p>
))}
```

aliceh75 replied:
Done ! Note I had to alias `name` to `institutionName` as otherwise it shadowed an outer variable.

creido-welly reviewed (approved):
One minor comment but all good otherwise

aliceh75 committed:
Use object destructuring when looping over objects.

Note the 'name' shadowed an outer variable, so an alias had to be used otherwise eslint complained.
